h2o-docs/src/product/data-munging/replacing-values.rst

```diff
@@ -17,12 +17,12 @@ This example shows how to replace numeric values in a frame of data. Note that i
         # so in the example below, the value in the 14th row and 2nd column will be set to 2.0.
```

Comment:
The R example uses (14th row, 2nd column) while the python example uses (15th row, 3rd column)